site/content/blog/incorporating-external-resources-in-kapp.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Incorporating external resources in kapp"
+slug: incorporating-external-resources-in-kapp
+date: 2021-12-24
+author: Praveen Rewar
+excerpt: "How to use kapp's exists annotation to wait for resources created by external agencies"
+image: /img/logo.svg
+tags: ['Praveen', 'kapp', 'exists']
+---
+
+kapp CLI encourages Kubernetes users to manage resources in bulk by working with "Kubernetes applications" (sets of resources with the same label). But there are often times when we want to incorporate resources that are not actually part of the same application (created by external agents). 
```

Comment:
However, there are real time use cases, when dependency needs to be incorporated in kapp resources that are not actually part of the same application i.e, they are created by external agents like CRDs.

Reply:
> they are created by external agents like CRDs.

It seems like we are saying that CRDs are the external agents here. Like the rest of it, so will update that :)